docs/mkdocs/docs/features/binary_formats/bson.md

```diff
@@ -94,3 +94,7 @@ The library maps BSON record types to JSON value types as follows:
     ```json
     --8<-- "examples/from_bson.output"
     ```
+
+!!! note "Handling of BSON type 0x11"
+BSON type 0x11 is used to represent uint64 numbers. This library treats these values purely as uint64 numbers 
```

Comment:
After a `!!!` block,

- please add an empty line
- indent the text below with 4 spaces

Only then, a box is rendered in the final documentation.